build-tools/cmake/xa_macros.cmake

```diff
@@ -51,10 +51,8 @@ macro(xa_common_prepare)
     fPIC
     )
 
-  if(CMAKE_BUILD_TYPE STREQUAL Debug)
-    set(XA_COMPILER_FLAGS ${XA_COMPILER_FLAGS} ggdb3 fno-omit-frame-pointer O0)
```

Comment:
Alternatively, could/should we have `bin/Debug/...` be built with `-ggdb3 -0)` and `bin/Release/...` built with `-O2`?

Commercial build outputs come from `bin/Release/...`, so this arrangement would mean that local (default `bin/Debug` builds would be more easily debuggable, while what our customers get would be optimized.)

Reply:
I wouldn't bother. We don't use any debuggers to debug our native code anyway. Let's keep things simple. And if we do need to use a debugger, it's easy to modify the cmake config locally to get unoptimized builds.